### ISSUE TITLE
CSS not loaded in the correct order

### ISSUE THREAD

**Issue by jgrenat** [geelen/jspm-loader-css#9]
Hi,

Yesterday, I tried to import some CSS files. In two of these files, I have rules with the same priority, something like : 

``` css
/* File 1 */
.my-class {
    background-color: red;
}

/* File 2 */
.my-class {
    background-color: green;
}
```

``` js
import 'file1.css';
import 'file2.css';
```

The expected result is to have a green background, but as my first file is heavier than my first, it's loaded after the second one, making my override fail.

**Comment by geelen** [comment]
Yes! This is a nasty bug, and I've fixed it once before. I suspected it wouldn't happen in JSPM but if you're seeing it, I'll go in and port the fix :)

**Comment by geelen** [comment]
Fixed by #12

**Comment by geelen** [comment]
And released in `v0.1.5`

**Comment by jgrenat** [comment]
I've just updated the version - it works perfectly! Thanks for your work :)
